apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/MetricOptions.java

```diff
@@ -0,0 +1,90 @@
+package gov.cms.bfd.pipeline.app;
+
+import java.io.Serializable;
+
+/** Models the user-configurable options for sending telemetry to New Relic. */
+public final class MetricOptions implements Serializable {
```

Comment:
You'll have a warning here that it needs a default version blah blah blah field, I think?

Reply:
Will look into it